third_party/codemirror-5.19.0/lib/codemirror.css

```diff
@@ -3,7 +3,8 @@
 .CodeMirror {
   /* Set height, width, borders, and global font properties here */
   font-family: monospace;
-  height: 300px;
+  height: 100%;
```

Comment:
Do not modify files in third-party, otherwise the versioning/pinning we're doing to ensure stability completely fails. We can add library CSS modifications in app.html for now, if needed.